rtlive/sources/data_be.py

```diff
@@ -1,39 +1,225 @@
 import logging
 import pandas
+import numpy
+import datetime
+
+from typing import Dict, Tuple, Union
 
-from . import ourworldindata
 from .. import preprocessing
 
 _log = logging.getLogger(__file__)
 
+# From https://en.wikipedia.org/wiki/Provinces_of_Belgium
+BE_REGION_NAMES = {
```

Comment:
The data workflow uses the region short codes (in this case 3-letter alphabetic) in all places.
This means we'll need these dicts to use the short codes as keys.